client/src/game/ui/HanabiCardMouse.ts

```diff
@@ -195,13 +195,17 @@ function shouldShowLookCursor(card: HanabiCard) {
   }
 
   // For ongoing games, only show the cursor for our hand if it has a custom card identity
-  if (
-    (card.note.suitIndex !== null &&
-      card.note.suitIndex !== card.state.suitIndex) ||
-    (card.note.rank !== null && card.note.rank !== card.state.rank) ||
-    card.note.blank ||
-    card.note.unclued
-  ) {
+
+  // Check if there exists a possibility from clues that the note declares impossible.
```

Comment:
punctuation

Reply:
you're killing me <3

Reply:
if you would do it right from the beginning then it wouldn't be my fault =p